hummingbot/strategy/pure_market_making/pure_market_making_config_map.py

```diff
@@ -120,6 +120,14 @@ def exchange_on_validated(value: str):
                   type_str="decimal",
                   validator=lambda v: validate_decimal(v, 0, 100, inclusive=False),
                   prompt_on_new=True),
+    "minimum_spread":
+        ConfigVar(key="minimum_spread",
+                  prompt="How close to the mid price do you want to cancel active orders? "
```

Comment:
"At what minimum spread should the bot automatically cancel orders? (Enter 1 for 1%)